package.json

```diff
@@ -42,6 +42,7 @@
     "css-loader": "^6.7.1",
     "cypress": "^10.4.0",
     "depcheck": "^1.4.3",
+    "dotenv": "^16.0.1",
```

Comment:
any reason for switching to this instead of envalid?

Reply:
works with webpack

Reply:
I don't think envalid works with webpack, this is a react app.

We could add envalid -> https://www.npmjs.com/package/envalid/v/6.0.2 but we are going to need dotenv eithere way. To be fair i like to keep deps to the minimun and envalid depends on `dotenv` and knowing that so far we have only morello-api thought we could either implement later or leave it